src/screens/flows/WalletConnect/screens/Flows/components/DappInfo/index.tsx

```diff
@@ -1,68 +1,48 @@
 import { t } from 'i18next';
 import React from 'react';
 import { Image, Text, View } from 'react-native';
-import { useSelector } from 'react-redux';
 
 import { FontStyles } from '@/constants/theme';
-import { State } from '@/interfaces/redux';
 import { FlowsRequest, WCFlowTypes } from '@/interfaces/walletConnect';
 
-import UserShowcase from '../UserShowcase';
+import unknownDappIcon from '../../assets/unknownIcon.png';
```

Comment:
i added this same asset as SVG in 0.2.0, we should add a TODO here to use that one when its merged